io.github.accessory.minus_games_client.yml

```diff
@@ -0,0 +1,439 @@
+id: io.github.accessory.minus_games_client
```

Comment:
the check fail because the name doesn't match the repo that is name `minus_games`

Reply:
Created new repo for this